cluster/charts/crossplane-types/crds/database.crossplane.io_postgresqlinstances.yaml

```diff
@@ -146,10 +146,12 @@ spec:
               type: object
             engineVersion:
               description: 'EngineVersion specifies the desired PostgreSQL engine
-                version. Allowed Versions: 9.6 and 11.'
+                version. Allowed Versions: 9.4, 10.0, 11.0 and 12.0.'
```

Comment:
@zzxwill just a heads up, if we want to modify this we would do so in the go code, then it will be generated here with `make generate`. That being said, we want to use the lowest common denominator values here across all providers, so we likely would not support all of these versions. This is something claims run into frequently, so they will likely be deprecated soon because composition provides a much more robust experience :)

Reply:
+1. Deprecation of the `engineVersion` values is great. On the one hand, composition provider will return more accurate exception information, on the other hand, these values diff from each those engine versions from Cloud Providers.